CHANGES.rst

```diff
@@ -4,6 +4,7 @@ Changes
 Version 0.9.0 (UNRELEASED)
 --------------------------
 
+- Fixes when Gitlab hook is received and user exceeds quota limit, commit build status is properly failed.
```

Comment:
The order of news items is usually Adds/Changes/Fixes, to announce new features before feature changes and bug fixes. (Although we don't always respect it everywhere!)  You can also reformulate the text to read like a user-oriented phrase, perhaps:

- Fixes GitLab integration error reporting in case user exceeds CPU or Disk quota usage limits.